src/legacy/server/i18n/localization/telemetry_localization_collector.test.ts

```diff
@@ -36,7 +36,7 @@ import { getTranslationCount } from './telemetry_localization_collector';
 describe('getTranslationCount', () => {
   it('returns 0 if no translations registered', async () => {
     const i18nLoaderMock = createI18nLoaderMock({});
-    const count = await getTranslationCount(i18nLoaderMock, 'en');
+    const count = await getTranslationCount(i18nLoaderMock as any, 'en');
```

Comment:
NIT: If `createI18nLoaderMock` cannot be adapted to return the proper signatures, I would at least have it handle the cast to `typeof i18nLoader` to avoid doing it in every test.